docs/api/utils.rst

```diff
@@ -0,0 +1,604 @@
+:meth:`RecoverSpanQuad`              compute the quad of a subset of span characters
+:meth:`Unicode2GlyphName`            return glyph name from a unicode
+:meth:`MakeAnnotDA`                  Passing color, fontname, fontsize into the annot
+:meth:`AddAnnotId`                   Add a unique /NM key to an annotation or widget
+:meth:`AddOcObject`                  Add OC object reference to a dictionary
+:meth:`ColorCount`                   Return count of each color.
+:meth:`BinFromBuffer`                Turn FzBuffer into a byte[]
+:meth:`BufferFromBytes`              Make FzBuffer from a byte[] object
+:meth:`CalcImageMatrix`              Compute image insertion matrix
+:meth:`CompressBuffer`               Compress FzBuffer into a new buffer
+:meth:`GetPdfString`                 Make a PDF-compatible string
+:meth:`ConstructLabel`               Construct a label based on style, prefix and page number
+:meth:`DecodeRawUnicodeEscape`       Decode raw unicode
+:meth:`DoLinks`                      Insert links contained in copied page range into destination PDF
+:meth:`EnsureIdentity`               Store ID in PDF trailer
+:meth:`ExpandFontName`               Make /DA string of annotation
+:meth:`GetId`                        Count numbers and return unique id on one process
+:meth:`GetAllContents`               All /Contents streams concatenated to one bytes object.
+:meth:`GetAnnotByName`               Retrieve annot by name (/NM key)
+:meth:`GetArea`                      Calculate area of rectangle
+:meth:`GetBorderStyle`               Return int meaning PdfObj "border style" from string type
+:meth:`GetColors`                    Retrieve the red, green, blue triple of a color name
+:meth:`GetColorHSV`                  Retrieve the hue, saturation, value triple of a color name
+:meth:`GetColorInfoList`             Returns Tuples containing of color name, red, green, blue color values
+:meth:`GetDestString`                Calculate the PDF action string
+:meth:`GetFieldTypeText`             Returns field type string from int type
+:meth:`GetFontProperties`            Returns properties of the font having xref in PDF
+:meth:`GetGlyphText`                 Adobe Glyph List function
+:meth:`GetImageExtension`            Return extension for MuPDF image type
+:meth:`GetLinkText`                  Define skeletons for /Annots object texts
+:meth:`GetWidgetProperties`          Populate a Widget object with the values from a PDF form field
+:meth:`InsertContents`               Insert a buffer as a new separate /Contents object of a page
+:meth:`Integer2Letter`               Return letter sequence string for integer i
+:meth:`Integer2Roman`                Return roman numeral for integer i
+:meth:`MeasureString`                Calculate the width of the text
+:meth:`MergeRange`                   Copy a range of pages (spage, epage) from a source PDF to a specified location (apage) of the target PDF
```

Comment:
No idea what `spage` , `epage` & `apage` are - I guess "Start Page" , "End Page" and "Actual Page" ? But I am guessing - can we explain more?